app/controllers/authenticate_controller.rb

```diff
@@ -123,6 +123,18 @@ def authenticator_input
     )
   end
 
+  def authenticator_input_without_credentials
```

Comment:
can you add a comment with the purpose of this method?

Reply:
Done